docs/deploy.md

```diff
@@ -0,0 +1,131 @@
+
+1.  Release AMY v1.X.Y and version bump on master
+
+    1.  Move to AMY root directory (the one with `manage.py` file) using `cd` command.
+    
+    1.  Make sure you have configured repositories:
+    
+        - `origin` for your repository on GitHub
+        - `upstream` for [`swcarpentry/amy` repo on GitHub](https://github.com/swcarpentry/amy)
+        
+        For example, this is the correct configuration for `chrismedrela`:
+        
+            $ git remote -v
+            origin	git@github.com:chrismedrela/amy.git (fetch)
+            origin	git@github.com:chrismedrela/amy.git (push)
+            upstream	git@github.com:swcarpentry/amy.git (fetch)
+            upstream	git@github.com:swcarpentry/amy.git (push)
+    
+    1.  Make sure your local `develop` and `master` branches are up to date:
+    
+            $ git checkout develop
+            $ git pull upstream develop
+            $ git push origin develop
+            
+            $ git checkout master
+            $ git pull upstream master
+            $ git push origin master
+        
+        Pushes to your `origin` repository are optional.
+        
+    1.  Update `master` branch:
+        
+            $ git checkout master
+            $ git merge --no-ff develop  # sometimes there are conflicts in files with release version, but most often we need to edit them and commit
+            
+    1.  Bump version on `master`:
+    
+            $ vim bower.json  # change 'version' to 1.X.Y
+            $ vim workshop/__init__.py  # change version to 1.X.Y
+            $ git add workshops/__init__.py bower.json
+            $ git commit -m "Bumping version to v1.X.Y"
+    
+    1.  Just to be safe, run tests.
+    
+            $ make test
+    
+    1.  Tag a release. Tag message should be `AMY release v1.X.Y`. 
+        
+            $ git tag -a v1.7.1 -s
+            
+        Omit `-s` flag if you cannot create signed tags.
+    
+    1.  Push `master` and the new tag everywhere:
+    
+            $ git push origin master --tags
+            $ git push upstream master --tags
+    
+1.  Bump version on `develop`:
+
+        $ git checkout develop
+        $ vim bower.json  # change version to the next one after 1.X.Y (i.e. 1.X+1.Y)
+        $ vim workshop/__init__.py  # change version to the next one
+        $ git add workshops/__init__.py bower.json
+        $ git commit -m "Bumping version to v1.8.0-dev"
+        
+    And push it everywhere:
+        
+        $ git push upstream develop
+        $ git push origin develop
+
+1.  Update production.
+
+    1.  Log into production:
+
+            ssh amy@amy.software-carpentry.org
+
+        All the following commands will be executed on production, not your local machine.
+        
+    1.  Execute:
+        
+            source ~/amy_site/venv/bin/activate
+            cd amy_site/amy
+            git fetch
+            git checkout tags/v1.7.1  # Python files updated
```

Comment:
"Python files updated, but not the served files - server keeps them in memory" or something similar